diff --git a/.htaccess b/.htaccess
index 9a73a3d3..7f81246a 100644
--- a/.htaccess
+++ b/.htaccess
@@ -69,12 +69,9 @@ AddEncoding gzip svgz
 <IfModule mod_rewrite.c>
   RewriteEngine on

-  # Set "protossl" to "s" if we were accessed via https://.  This is used later
-  # if you enable "www." stripping or enforcement, in order to ensure that
-  # you don't bounce between http and https.
-  RewriteRule ^ - [E=protossl]
-  RewriteCond %{HTTPS} on
-  RewriteRule ^ - [E=protossl:s]
+# Rewrite http://www.example.com to https://www.example.com
+  RewriteCond %{HTTPS} off
+  RewriteRule ^ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]

   # Make sure Authorization HTTP header is available to PHP
   # even when running as CGI or FastCGI.
